src/Apps/W1/External Storage - Document Attachments/app/src/Permissions/DAExtStorAdmin.PermissionSet.al

```diff
@@ -0,0 +1,17 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.ExternalStorage.DocumentAttachments;
+
+/// <summary>
+/// Permission set for External Storage functionality.
+/// Grants necessary permissions to use external storage features.
+/// </summary>
+permissionset 8751 "DA Ext. Stor. Admin"
```

Comment:
Was discussing the permissions with @JesperSchulz and we think it should be slightly different.
Basically, all permissions should be inherent, except for the tabledata "DA External Storage Setup" which only an admin should be able to RIMD.
Everyone else still needs little r, so that should be inherent and then you would need to also add little r in code wherever the table is read.